llvm/unittests/Analysis/CGSCCPassManagerTest.cpp

```diff
@@ -1633,8 +1633,7 @@ TEST_F(CGSCCPassManagerTest, TestUpdateCGAndAnalysisManagerForPasses8) {
 
         // Create an unsused constant that is referencing the old (=replaced)
```

Comment:
```suggestion
        // Create an unused constant that is referencing the old (=replaced)
```
Fix this typo while you're here?